builds/running-entitled-builds.adoc

```diff
@@ -0,0 +1,14 @@
+:context: running-entitled-builds
+[id='running-entitled-builds_{context}']
+= Running entitled builds
+include::modules/common-attributes.adoc[]
+toc::[]
+
+Use the following sections to run entitled builds on {product-title}.
+
+include::modules/builds-source-input-satellite-config.adoc[leveloffset=+1]
+include::modules/builds-source-input-subman-config.adoc[leveloffset=+1]
+include::modules/builds-source-secrets-entitlements.adoc[leveloffset=+1]
+include::modules/builds-strategy-docker-entitled-satellite.adoc[leveloffset=+1]
+include::modules/builds-strategy-docker-entitled-subman.adoc[leveloffset=+1]
+include::modules/builds-strategy-docker-squash-layers.adoc[leveloffset=+1]
```

Comment:
I'd want the tree to be as follows:

0. Create an ImageStreamTag to registry.access.redhat.io/rhel7/rhel:latest (an admin can do this and place the IS in the openshift namespace)
1. Set up entitlement secret
2. Subscription Manager
  a. Source input - subman
  b. Dockerfile - subman
3. Satellite
  a. Source input - satellite
  b. Dockerfile - satellite
4. Squash layers

Reply:
Per conversation with @bparees we will also want to include the module on how to add a pull secret. So there are two paths to pulling in the base RHEL image:
1. Add the pull secret to registry.redhat.io to your project
2. Have an admin create an ImageStream in the `openshift` namespace for the RHEL base image - this makes the IS available across the cluster.